libs/extensions/include/NanoEvent.hpp

```diff
@@ -36,7 +36,11 @@ class NanoEvent {
   float GetNDSAMuon(std::string collectionName);
   float GetNMuon(std::string collectionName);
 
-  std::shared_ptr<PhysicsObject> GetMuonWithIndex(int muon_idx, std::string collectionName, bool isDSAMuon);
+  std::shared_ptr<PhysicsObject> GetMuonWithIndex(int muon_idx, std::string collectionName, bool isDSAMuon=false);
+  std::pair<float,int> GetMinDeltaRToGenMuons(std::shared_ptr<PhysicsObject> recoMuon);
```

Comment:
This returns a float and int - I can guess the float is DeltaR from the name, but I have no idea what this int is. Maybe something like `GetDeltaRandIndexOfClosestGenMuon`?